template-tags.php

```diff
@@ -77,7 +77,7 @@ function is_coauthor_for_post( $user, $post_id = 0 ) {
 	return false;
 }
 
-class CoAuthorsIterator {
+class CoAuthors_Iterator {
```

Comment:
We can't change this one because of back-compat (if someone is using the class directly, for example).